apps/site/assets/.eslintrc.js

```diff
@@ -18,7 +18,7 @@ const baseConfig = {
     "react/require-default-props": "off",
     "prefer-object-spread": "off",
     "react/jsx-props-no-spreading": "off",
-    "react/no-unused-prop-types": "warn",
+    "react/no-unused-prop-types": "off",
```

Comment:
why the switch from `warn` to `off`? Too much noise?

Reply:
Thanks for flagging... looking at it again it's only five errors, maybe it's worth fixing manually so that I can leave this as "warn" 🤔

Reply:
@alicia-mbta I've restored it back to "warn" and updated a few components so that it would pass.